qa/L0_infer/test.sh

```diff
@@ -62,6 +62,16 @@ if [ "$TENSORRT_SERVER_CPU_ONLY" == "1" ]; then
     fi
 fi
 
+# If BACKENDS not specified, set to all
+if [ -z "$BACKENDS" ]; then
```

Comment:
Can you use bash syntax like this?

BACKENDS=${BACKENDS:="graphdef savedmodel netdef onnx libtorch plan custom"}

Reply:
Done